src/components/Table/Table.js

```diff
@@ -1,75 +1,63 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
-
 // @CHILD CLASSES
 import TableHeader from './TableHeader';
 import TableBody from './TableBody';
-
 class Table extends Component {
-
     constructor() {
         super();
-
         this.state = {
             activeRowID: [],
-            myUserList: []
+            myUserList: [],
+            searchValue: 0
         }
-
     }
-
-
     componentWillReceiveProps(nextProps) {
         this.setState({
             myUserList: nextProps.data
         });
     }
-
-    // (1) TODO HOME WORK
     deleteFromTable = () => {
         const array = this.state.myUserList;
         const activeRows = this.state.activeRowID;
-
         for (let i = 0; i < array.length; i++) {
             if (activeRows.includes(array[i])) {
                 delete array[i];
             }
-
         };
-
         this.setState({
             myUserList: array
         });
-
-
     }
-
+    searchFromTable = (e) =>{
```

Comment:
make searchFromTable handler in App Component and then path it via props to future InputSearchComponent